dev-tools/xtask/src/virtual_hardware.rs

```diff
@@ -90,13 +90,16 @@ pub struct Args {
     #[clap(long, default_value = "/var/tmp")]
     vdev_dir: Utf8PathBuf,
 
+    /// Allow customization of the vdev size
+    #[clap(long, default_value_t = 20 * GB)]
+    vdev_size: u64,
+
```

Comment:
1. This argument is only used in (and only makes sense in) the context of creating virtual hardware, so it should be part of the `Commands::Create` variant.
2. I would reword the doc comment as it's used to create the `--help` output. Perhaps something like `Size in bytes for created vdevs`.

Reply:
I have pushed the fmt update and moved the argument to the proper variant. No need for you to take time to do that. Thanks for the feedback!